python/lsst/daf/butler/registries/sql.py

```diff
@@ -219,6 +221,10 @@ def __init__(self, database: Database, defaults: RegistryDefaults, managers: Reg
         # initialized, and must be done before the property getter is used.
         self.defaults = defaults
 
+        # TODO: This is currently initialized by `make_datastore_tables`,
+        # eventually we'll need to do it during construction.
+        self._datastore_record_classes: Mapping[str, type[StoredDatastoreItemInfo]] = {}
```

Comment:
Are keys here datastore names or opaque table names?

I also wonder if it'd be better to just have a mapping of `DatastoreOpaqueTable`.

Reply:
Keys are opaque table names, I'll update comment. Registry itself does not need `TableSpec` part of `DatastoreOpaqueTable`, depending on how we initialize this structure in the future it may be better not to require `TableSpec` at that point.